indra/llui/llscrollcontainer.cpp

```diff
@@ -126,6 +126,7 @@ LLScrollContainer::LLScrollContainer(const LLScrollContainer::Params& p)
 	
 	LLRect horizontal_scroll_rect = mInnerRect;
 	horizontal_scroll_rect.mTop = horizontal_scroll_rect.mBottom + scrollbar_size;
+	horizontal_scroll_rect.mLeft += 3;
```

Comment:
'3' in this case is a 'magic number', please use a const variable, like HORIZONTAL_SCROLL_LEFT_PAD.

Reply:
On screen it looks like horizontal scroll overlaps vertical scroll now, are you shure you don' need to reduce with by the same 3 pixels?